recipes/configs/phi3/mini_lora_single_device.yaml

```diff
@@ -52,8 +52,8 @@ shuffle: True
 # Fine-tuning arguments
 epochs: 1
 max_steps_per_epoch: null
-gradient_accumulation_steps: 64
 batch_size: 2
+gradient_accumulation_steps: 64
```

Comment:
64 feels so high

Reply:
16?

Reply:
i think that Salmon also changed this default in some other config too.